includes/Exception.php

```diff
@@ -609,13 +610,6 @@ protected static function report( $e ) {
 			} else {
 				self::escapeEchoAndDie( $message );
 			}
-
-			# Wikia change - begin
-			# @see PLATFORM-2008 - report non-MediawWiki exceptions to ELK
-			Wikia\Logger\WikiaLogger::instance()->error( __METHOD__ . ' - unexpected non-MediaWiki exception encountered', [
-				'exception' => $e,
-			] );
-			# Wikia change - end
```

Comment:
`self::escapeEchoAndDie` dies internally when run on production - hence we got no logs for non-MediaWiki exceptions...